agent/lib_phpunit.c

```diff
@@ -665,6 +666,22 @@ static int nr_phpunit_are_statuses_valid(TSRMLS_D) {
   return 1;
 }
 
+void nr_phpunit_version() {
+  char* string = "PHPUnit\\Runner\\Version::id();";
```

Comment:
Same const suggestion as https://github.com/newrelic/newrelic-php-agent/pull/775/files#r1440962256

Reply:
After discussion, this will not be added. Same reasoning as: https://github.com/newrelic/newrelic-php-agent/pull/775#discussion_r1442174308